tiledb/sm/config/config.h

```diff
@@ -102,6 +102,9 @@ class Config {
   /** The default for Curl's verbose mode used by REST. */
   static const std::string REST_CURL_VERBOSE;
 
+  /** If the array enumerations should be loaded on array open */
+  static const std::string REST_LOAD_ENUMERATIONS_ON_ARRAY_OPEN;
```

Comment:
Sanity question: everything works as expected if client and rest server have this option set differently (false,true) or (true,false) ?

Reply:
It would if they were different, but the config used by the REST server is provided by the client so they can't actually be different (given the way things are implemented currently).